CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v7.7.1 (24. February 2021)
+
+### Fix
+* Extract_global_id should not be too eager in decoding ([#1389](https://github.com/winged/caluma/issues/1389)) ([`8f97dc6`](https://github.com/winged/caluma/commit/8f97dc6ef30684653d1eba8036606d28386e554a))
```

Comment:
```suggestion
* Extract_global_id should not be too eager in decoding ([#1389](https://github.com/projectcaluma/caluma/issues/1389)) ([`8f97dc6`](https://github.com/winged/caluma/commit/8f97dc6ef30684653d1eba8036606d28386e554a))
```

That's a weird behavious that came with a version of semantic-release. I never got the chance to follow it - but it needs to be fixed

Reply:
HMM wth.. fixing